src/audio/dcblock/dcblock_hifi4.c

```diff
@@ -10,7 +10,7 @@
 
 #include "dcblock.h"
 
-#ifdef DCBLOCK_HIFI4
+#if SOF_USE_HIFI(4, DCBLOCK)
```

Comment:
@andrula-song or anyone working with HiFi5 please make/submit this temporary change until dcblock_hifi5.c exists:
```suggestion
#if SOF_USE_HIFI(4, DCBLOCK) || SOF_USE_HIFI(5, DCBLOCK)
```